frontend/config-assistant/src/main.ts

```diff
@@ -33,4 +33,9 @@ app.config.errorHandler = (error: unknown) => errorService.onError(error);
 
 registerIcons();
 
+// warn the user if he closes the app
+window.addEventListener('beforeunload', event => {
+  event.returnValue = null;
```

Comment:
is this necessary? is also working without this line? Because it is confusing why we set this to null

Reply:
yes this is necessary, without this it is not working, we have to defined returnValue with null or some message, its because of some different browser. it seems like if any other browser don't have their own message then it uses what we here defined. may be I thought, but yeah it is necessary either null or with some messages